src/storage/constants.tsx

```diff
@@ -7,6 +7,8 @@ import {
   Values,
 } from './types';
 
+export const storeVersionKey: string = 'storeVersion';
```

Comment:
It might be a better name, but we have used "store" in many other places. I also think it is fairly common to refer to the datastorage as store in React applications.